### DISCUSSION TITLE
Is it possible to separate the UI of flood on desktop and mobile?

### DISCUSSION THREAD

**Discussion by lilws** [jesec/flood#192] [Show and tell]
I mainly use flood for mobile, but since it offers a lot of full features of torrent client. I find that Condensed view is eay to monitor on desktop, and on mobile the expanded view is comfortable and easy to navigate. Can we have a option of auto switch between the 2 UI for desktop and mobile?

**Comment by jesec** [comment]
It is on the roadmap.

https://github.com/jesec/flood/projects/4#card-44000260
